### PR TITLE
accordion on fire

### PR DESCRIPTION
**Pull request by hermsicle**

### CODE REVIEW

File: src/components/Accordion/index.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useState, createContext, useContext } from 'react'
 import { Container, Heading, Inner, Frame, Header, Title, Toggle, Body, Content } from './_Accordion.styles'
 import { Plus, Close } from '../Icon/'
+import data from '../../json/_faq.json'
```
Comment (awwmicky) [inline src/components/Accordion/index.jsx:4]: change to `import { faq } from '../../json'`

File: src/components/Accordion/index.jsx [hunk 2]
```diff
@@ -18,34 +19,50 @@ Accordion.Inner = function AccordionInner ({ children, ...restProps }) {
...
-	const [ toggle, setToggle ] = useState(false)
+	const [ currentIndex, setCurrentIndex ] = useState()
+	const [ isOpen, setIsOpen ] = useState(false)
 
 	return (
-		<ToggleContext.Provider value={{ toggle, setToggle }}>
-			<Frame { ...restProps }> { children } </Frame>
-		</ToggleContext.Provider>
+		<>
+			{
```
Comment (awwmicky) [inline src/components/Accordion/index.jsx:27]: combine Ln. 27 + 28 `{ data.map...`
combine Ln. 34 + 35 `))}`

File: src/containers/FaqContainer/index.jsx [hunk 3]
```diff
@@ -1,32 +1,24 @@
...
-import { Plus } from '../../components/Icon/'
 
 const FaqContainer = () => {
 	return (
 		<Accordion>
 			<Accordion.Heading>Frequently Asked Questions</Accordion.Heading>
 			<Accordion.Inner>
-				{
-					data.map( ( acc, i ) => (
-						<Accordion.Frame key={ i }>
```
Comment (awwmicky) [inline src/containers/FaqContainer/index.jsx:13]: for the key, use `import { v4 as uuid } from 'uuid'`

**Review by awwmicky — CHANGES_REQUESTED** [review]
make some subtle changes for consistency

File: src/containers/FaqContainer/index.jsx (hunk 3 above)
Comment (hermsicle) [reply src/containers/FaqContainer/index.jsx:13]: uuid throws a whole bunch of errrors on console

**Pull request merged**